src/renderer/renderElement.ts

```diff
@@ -229,9 +272,21 @@ function generateElement(
       case "arrow": {
         const options: Options = {
           stroke: element.strokeColor,
-          strokeWidth: element.strokeWidth,
-          roughness: element.roughness,
+          strokeWidth:
+            element.strokeStyle !== "solid"
+              ? element.strokeWidth + 0.5
+              : element.strokeWidth,
           seed: element.seed,
+          // we need to add dashed stroked for arrows manually outside rough,
+          //  because we want to prevent dashed stroke for arrow points
+          strokeLineDash:
+            element.type === "line" && element.strokeStyle === "dashed"
+              ? DASHARRAY_DASHED
+              : element.type === "line" && element.strokeStyle === "dotted"
+              ? DASHARRAY_DOTTED
+              : undefined,
+          roughness: element.roughness,
+          disableMultiStroke: element.strokeStyle !== "solid",
```

Comment:
Not sure when you need to manually do this here. When drawing the arrow head, do not set the line dash option in roughjs command.

Reply:
Later in this method, under the comment "add lines only in arrow", the options you pass to the `generator.line` function, unset the `strokeLineDash` option

Reply:
You're right. I wasn't aware we're drawing those manually.